### PR TITLE
Added skip wallpaper flag

### PR DESCRIPTION
**Pull request by hallgat89**
This is a flag to skip setting the wallpaper. Why? Wal sets every wallpaper in "filled" mode, even if I would like to use a tiled pattern instead. With this flag I disable the wallpaper, and set the background myself.

### CODE REVIEW

File: wal [hunk 1]
```diff
@@ -163,46 +163,52 @@ send_sequences() {
...
-        *"MUFFIN"* | *"Cinnamon"*) gsettings set org.cinnamon.desktop.background picture-uri "file://${wal}" ;;
-        *"MATE"*) gsettings set org.mate.background picture-filename "$wal" ;;
-        *"GNOME"*) gsettings set org.gnome.desktop.background picture-uri "file://${wal}" ;;
-
-        *"XFCE"*)
-            xfconf-query --channel xfce4-desktop --property /backdrop/screen0/monitor0/image-path --set "$wal" 2>/dev/null ||\
-            xfconf-query --channel xfce4-desktop --property /backdrop/screen0/monitor0/workspace0/last-image --set "$wal" 2>/dev/null
-        ;;
+    
+    if [[ "$nowall" != "on" ]]
```
Comment (dylanaraps) [inline wal]:
```sh
if [[ -z "$nowall" ]]; then
```

File: wal [hunk 2]
```diff
@@ -163,46 +163,52 @@ send_sequences() {
...
+                else
+                    gsettings set org.gnome.desktop.background picture-uri "file://${wal}"
+                fi
+            ;;
+        esac
 
-    out "wallpaper: Set new wallpaper"
+        out "wallpaper: Set new wallpaper"
+    else
+        out "skipping wallpaper"
```
Comment (dylanaraps) [inline wal]:
```sh
out "wallpaper: '-n' was used, skipping wallpaper"
```

File: wal [hunk 3]
```diff
@@ -320,7 +327,7 @@ Flags:
 }
 
 get_args() {
-    while getopts ":a:chi:o:qrt" opt; do
+    while getopts ":a:chi:o:qrtn" opt; do
```
Comment (dylanaraps) [inline wal]:
```sh
while getopts ":a:chi:no:qrt" opt; do
```

File: wal [hunk 4]
```diff
@@ -335,6 +342,7 @@ get_args() {
             "q") quiet="on" ;;
             "r") reload="on" ;;
             "t") vte="on" ;;
+            "n") nowall="on" ;;
```
Comment (dylanaraps) [inline wal]: Move this so that it's in alphabetical order with the other args.

**Review by dylanaraps — CHANGES_REQUESTED** [review]

**Review by hallgat89 — COMMENTED** [review]
It's okay for me :)

**Comment by dylanaraps** [comment]
It may work fine but it's not yet up to my standards. The PR only needs a few changes before it matches the style of the rest of the script.

I won't merge this until the above changes are made.

File: wal (hunk 1 above)
Comment (hallgat89) [reply wal]: done

File: wal (hunk 2 above)
Comment (hallgat89) [reply wal]: done

File: wal (hunk 3 above)
Comment (hallgat89) [reply wal]: Done, and ordered

File: wal (hunk 4 above)
Comment (hallgat89) [reply wal]: done and ordered

**Review by hallgat89 — COMMENTED** [review]
updated

File: wal [hunk 5]
```diff
@@ -330,12 +337,14 @@ get_args() {
...
             ;;
-
+            
+            "n") nowall="on" ;;
             "o") external_script=("$OPTARG") ;;
             "q") quiet="on" ;;
             "r") reload="on" ;;
             "t") vte="on" ;;
 
+
```
Comment (dylanaraps) [inline wal]: Remove this blank line.

File: wal [hunk 6]
```diff
@@ -163,46 +163,52 @@ send_sequences() {
 }
 
 set_wallpaper() {
-    # Get desktop environment.
-    de="${XDG_CURRENT_DESKTOP}"
+
```
Comment (dylanaraps) [inline wal]: Remove this blank line.

**Review by dylanaraps — COMMENTED** [review]
Once this is done, I'll merge.

Thanks for the PR! :)

File: wal (hunk 6 above)
Comment (hallgat89) [reply wal]: done

File: wal (hunk 5 above)
Comment (hallgat89) [reply wal]: done

**Comment by dylanaraps** [comment]
LGTM :+1:

Thanks for the PR!

**Review by dylanaraps — APPROVED** [review]

**Comment by hallgat89** [comment]
Thanks for the cooperation :)

**Pull request merged**